elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchJson.java

```diff
@@ -265,12 +265,69 @@ Duration took() {
       return this.hits;
     }
 
-    public long total() {
+    public SearchTotal total() {
       return total;
     }
 
   }
 
+  /**
+   * Container for total hits
+   */
+  @JsonDeserialize(using = SearchTotalDeserializer.class)
+  static class SearchTotal {
+
+    private final long value;
+    private final String relation;
+
+    SearchTotal(final long value, final String relation) {
+      this.value = value;
+      this.relation = relation;
```

Comment:
Is this field (`relation`) used anywhere ? Do we need it ?

Reply:
`relation` is not used anywhere. I removed it.